src/components/EventComponent/EventComponent.js

```diff
@@ -30,9 +30,9 @@ const EventComponent = ({ message }) => {
 
   if (
     type === 'channel.event' &&
-    (event.type === 'member.removed' || event.type === 'member.added')
+    (event?.type === 'member.removed' || event?.type === 'member.added')
```

Comment:
not sure if we should use all these optional chainings just to silence the ts compiler

Reply:
Moved to just a single event check, but I think this is needed until we are in real TS, then we could cast or type guard based on the `type` but in this JSDocs format it get's messy quick.